### ISSUE TITLE
✨ Fix Extension Buttons

### ISSUE THREAD

**Issue by rahaaatul** [yodaluca23/HomeWhereYouBelong#2]
![image](https://github.com/user-attachments/assets/422ca1e3-31c2-4a4f-aabc-4e56a6cada6b)

These extensions are in use:
```ini
extensions            = bookmark.js|popupLyrics.js
```

Version info:
```
Spotify for Windows (64 bit)
1.2.52.442.g01893f92
Spicetify v2.38.5
xpui_2024-12-02_1733143341811_01893f9
cef_130.1.16+g5a7e5ed+chromium-130.0.6723.117
Runtime: Chrome
Build System: Bazel
```

**Comment by yodaluca23** [comment]
Whats the problem?

**Comment by rahaaatul** [comment]
Requested for another script to fix those buttons.

**Comment by yodaluca23** [comment]
I've taken a crack at this, I don't think, at least the way I'm doing it, is very viable, theirs too much variation between each extension's buttons, and they reload every page update. Not to mention basically every theme I tried brought new bugs, I got it mostly working with Default Dynamic (the theme I use), but there were still some crucial visual bugs I could not find solutions for, if someone else wants to take a try at this, I'm willing to share what I've come up with. For now this will be closed.
